core/bentley/src/TupleKeyedMap.ts

```diff
@@ -0,0 +1,104 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Collections
+ */
+
+/** the set of Map interface functions supported by TupleKeyedMap */
+interface PartialMap<K, V> {
+  get(k: K): V | undefined;
+  set(k: K, v: V): PartialMap<K, V>;
+  has(k: K): boolean;
+  size: number;
+  [Symbol.iterator](): IterableIterator<[K, V]>;
+}
+
+/** A map similar to the standard JavaScript Map collection except that the keys must be a tuple
+ * (javascript array), and value comparison is used on these tuple keys, without a user-provided hash function.
+ * This means you can use array literals to store complicated data.
+ * ```js
+ * const map = new TupleKeyedMap([[1,"y"], "value"]);
+ * const value = map.get([1, "y"]); // a normal map would identify these keys as different because they are independent objects!
+ * ```
+ * It is implemented by each index of the tuple key being used as a singular key into a submap
+ * @note this only implements a subset, [[PartialMap]], of the Map interface
```

Comment:
`[[PartialMap]]` will be reported as invalid link by docs generator because `PartialMap` is not exported.

Reply:
removed, going to run the docs build locally to check for other bad links